views/templates/admin/marketing_step0/buy_step0.tpl

```diff
@@ -10,84 +10,71 @@
-									<span style="width:170px; display:inline-block">
-										{if isset($ticket.promo_price) and ($ticket.promo_price > 0)}
-											<strike>{$ticket.normal_price|number_format:2:",":"."} {l s='€' mod='expressmailing'}</strike>
-											&nbsp;
-											<font color="red"><b>{$ticket.promo_price|number_format:2:",":"."} {l s='€' mod='expressmailing'}</b></font>
-											<br>{l s='(Let %.3f € / sms)' mod='expressmailing' sprintf=($ticket.promo_price / $ticket.product_units)}
-										{else}
-											<b>{$ticket.normal_price|number_format:2:",":"."} {l s='€' mod='expressmailing'}</b><br>{l s='(Let %.3f € / sms)' mod='expressmailing' sprintf=($ticket.normal_price / $ticket.product_units)}
-										{/if}
-									</span>
-								</label>
-							</div>
-						{/foreach}
-					</div>
-				{/if}
-			</div>
-		</div>
-		<!-- /.form-wrapper -->
-		<div class="panel-footer">
-			<button type="submit" value="1" name="submitBuyStep0" class="btn btn-default pull-right"><i class="process-icon-cart"></i>{l s='Add to Cart' mod='expressmailing'}</button>
-		</div>
-	</div>
-</form>
-{/if}
\ No newline at end of file
+            <i class="icon-cogs"></i>&nbsp;{l s='Loss leader pricing' mod='expressmailing'}
+            {if !empty($tool_tip)}
+                <span class="panel-heading-action">
+                    <a class="list-toolbar-btn" href="#">
+                        <span class="label-tooltip" data-placement="left" data-html="true" data-original-title="{$tool_tip|escape:'html':'UTF-8'}" data-toggle="tooltip" title="">
+                            <i class="process-icon-help"></i>
+                        </span>
+                    </a>
+                </span>
+            {/if}
+        </div>
+        <div class="form-wrapper">
+            <div class="form-group">
+                <div>
+                    <table class="media_choice">
+					<tr style="height: 25px">
+						<td class="colorcell"><div style="background-color: rgb(148, 190, 42);"></div></td>
+						<td class="linkcell">{l s='Emailing' mod='expressmailing'}</td>
+						<td class="commentcell" style="vertical-align: middle; padding-top: 3px">
+							{l s='Up to %d free email per day' mod='expressmailing' sprintf=$broadcast_max_daily}
+							<b>&nbsp;-&nbsp;{l s='ou' mod='expressmailing'}&nbsp;-&nbsp;</b>
+							<a id="em_bying_link_email2"{if $smarty_email_disabled} disabled="disabled"{/if}>{l s='Sign Up for a Premium Plan' mod='expressmailing' sprintf=$smarty_email_lowest_price}</a>
+							{if !empty($smarty_email_promotion)}
+								<span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
+							{/if}
+						</td>
+					</tr>
+					<tr style="height: 25px">
+						<td class="colorcell"><div style="background-color: rgb(199, 111, 143);"></div></td>
+						<td class="linkcell">{l s='Fax' mod='expressmailing'}</td>
+						<td class="commentcell" style="vertical-align: middle; padding-top: 3px">
```

Comment:
This style definition is duplicated. If you wrote it in a CSS file, you just had to add a class.